specification/riskiq/data-plane/Microsoft.Easm/preview/2022-09-01-preview/easm.json

```diff
@@ -0,0 +1,5550 @@
+          },
+          "type": "array"
+        },
+        "technicalPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AssetId": {
+      "description": "The list of excludes used for the disco group runs, aka assets to exclude from the discovery algorithm.",
+      "properties": {
+        "displayName": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "kind": {
+          "enum": [
+            "as",
+            "contact",
+            "domain",
+            "host",
+            "ipAddress",
+            "ipBlock",
+            "page",
+            "sslCert"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "assetIdKind"
+          }
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "AssetLabelRequest": {
+      "properties": {
+        "transfers": {
+          "enum": [
+            "as",
+            "contact",
+            "domain",
+            "host",
+            "ipAddress",
+            "ipBlock",
+            "page",
+            "sslCert"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "assetLabelRequestTransfers"
+          }
+        },
+        "values": {
+          "additionalProperties": {
+            "type": "boolean"
+          },
+          "type": "object"
+        }
+      },
+      "type": "object"
+    },
+    "AssetPageResponse": {
+      "properties": {
+        "content": {
+          "items": {
+            "$ref": "#/definitions/AssetResponse"
+          },
+          "type": "array"
+        },
+        "first": {
+          "type": "boolean"
+        },
+        "last": {
+          "type": "boolean"
+        },
+        "mark": {
+          "type": "string"
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "number": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "numberOfElements": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "size": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "totalElements": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "totalPages": {
+          "format": "int32",
+          "type": "integer"
+        }
+      },
+      "type": "object"
+    },
+    "AssetResponse": {
+      "properties": {
+        "auditTrail": {
+          "items": {
+            "$ref": "#/definitions/AuditTrailItem"
+          },
+          "type": "array"
+        },
+        "createdDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "discoGroupName": {
+          "type": "string"
+        },
```

Comment:
This changed from `discoGroup` and `discuRun` in 2022-04-01-preview. Intentional? Since both are previews, though, that's fine.

Reply:
Yes ... structural changes based on feedback.